1-js/11-async/03-promise-chaining/article.md

```diff
@@ -199,21 +181,19 @@ loadScript("/article/promise-chaining/one.js").then(script1 => {
 });
```

Comment:
You forgot to translate the comment on line 175:
` // this function has access to variables script1, script2 and script3`